docs/wiki/deployment/performance-safety.md

```diff
@@ -59,6 +59,49 @@ Profiling query: SELECT DISTINCT process.name, listening.port, listening.protoco
  D:2  C:1  M:0  F:0  U:2  processes_binding_to_ports (1/1): duration: 1.02116107941 cpu_time: 0.668809664 memory: 6340608 fds: 5 utilization: 44.3
 ```
 
+### Understanding the output from profile.py
+The osquery `profile.py` script uses `utils.py` in `tools/tests/` which uses python’s `psutil` library to collect process stats for osqueryi as its running given queries. 
+
+The script returns 5 stats:
+
+**Utilization (U)**: Utilization is calculated by taking the average of non-0 results of the cpu_percent(interval=1) function in `psutils.Process()`. This value can be greater than 100% for processes with threads running across different CPUs. The script sets an interval of 1 meaning that the function compares process time to system CPU times elapsed before and after the 1 second interval. This is a blocking call. 
+
+**CPU time (C)**: CPU time uses the `psutils.Process()`'s `cpu_times()` function. It returns a named tuple containing user, system, children_user, system_user, and iowait
+
+- user: time spent in user mode.
+- system: time spent in kernel mode.
+- children_user: user time of all child processes (always 0 on Windows and macOS).
+- system_user: user time of all child processes (always 0 on Windows and macOS).
+- iowait: (Linux) time spent waiting for blocking I/O to complete. This value is excluded from user and system times count (because the CPU is not doing any work).
+
+The profile script adds user and system together for the CPU 
+Time output.
+
+**Duration (D)**:
+Duration is calculated by taking the subtracting `start_time` - 2 from the current time. The start time is set before the script starts the `osqueryi` process to run the query. The `start_time` - 2 comes from the `--profile_delay` flag used by the profile.py script. This flag causes osquery to wait before and after running the code under test. The 2 is to make up for this wait time. 
+
+**fds (F)**: Uses the `num_fds()` function and returns the file descriptors used by the `osqueryi` process during query execution
+
+**Memory (M)**: Uses the `memory_info_ex()` function which is deprecated. psutils documentation suggests using `memory_info()` instead. The function returns a named tuple and the script uses the `rss` value in the tuple. RSS stands for resident set size and is the non-swapped physical memory used by the process. This should match the RES column in `top`.
+
+**Profile.py Simple Thresholds**
```

Comment:
should this be an h3 (`###`) instead of bold? I'm not really sure what it means

Reply:
I'll write a clearer title and use h3